apps/preview/src/App.tsx

```diff
@@ -1,10 +1,10 @@
-import { Button, ThemeProvider } from '@boolti/ui';
+import { Button, TextButton, ThemeProvider } from '@boolti/ui';
 
 const Icon = () => (
   <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
     <path
-      fillRule="evenodd"
-      clipRule="evenodd"
+      fill-rule="evenodd"
+      clip-rule="evenodd"
```

Comment:
```suggestion
      fillRule="evenodd"
      clipRule="evenodd"
```

아마 warning이 뜰 것 같습니다요

Reply:
아 요거 push 안했구나